typescript/web/src/components/website/Movie/movie.tsx

```diff
@@ -0,0 +1,46 @@
+import { Box, Heading, Flex, Center, AspectRatio } from "@chakra-ui/react";
+import * as React from "react";
+
+export const Movie = () => {
+  return (
+    <Box id="Movie" as="section">
+      <Box>
+        <Box
+          maxW="2xl"
+          mx="auto"
+          px={{ base: "6", lg: "8" }}
+          py={{ base: "16", sm: "20" }}
+          textAlign="center"
+        >
+          <Heading size="xl" mb="4" fontWeight="extrabold">
+            1 min product video tour to get you started
+          </Heading>
+
```

Comment:
This block can be moved in a `Header` component declared on top of this file

```suggestion
          <Header />
```

Reply:
Done